src/Components/AddProject.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+
+class AddProject extends Component {
+  constructor(){
+      super();
+      this.state = {
+        newProject: {}
+      }
+  }
+
+  static defaultProps = {
+    categories: ['Web Design', 'Web Development', 'Mobile Development']
+  }
+  handleSubmit(e){
+      if(this.refs.title.value === ''){
+          alert('Title is required');
+      } else {
+          this.setState({newProject: {
+            title: this.refs.title.value,
+            category: this.refs.category.value
+          }
+        }, function(){
+          //console.log(this.state);
+          this.props.addProject(this.state.newProject);
+        })
+      }
+      e.preventDefault();
+  }
+
+  render() {
+    let categoryOptions = this.props.categories.map(category => {
```

Comment:
you can use `const` here instead of `let`

Reply:
I will try it out. just come to know the difference again.
https://wesbos.com/let-vs-const/